testsuite/features/secondary/min_bootstrap_ssh_key.feature

```diff
@@ -1,8 +1,10 @@
 # Copyright (c) 2021-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
 #
-# This feature is a dependency for:
-# - features/secondary/min_bootstrap_script.feature : Due to the registration of a SLES Minion
+#
+# This feature can cause failures in the following features:
+# - features/secondary/min_ssh_tunnel.feature
```

Comment:
I would also add
- min_bootstrap_script.feature
- min_activationkey.feature

See https://github.com/SUSE/spacewalk/issues/21386

Reply:
It should be `min_bootstrap_script.feature` here according to the running order, good catch
However for `min_activationkey.feature` I would argue it would fail only if `features/secondary/min_ssh_tunnel.feature` fails. If this feature fails and the following one manages to bootstrap again the minion, the cascade would stop there.